app/src/main/java/org/sopt/at/viewmodel/AuthViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package org.sopt.at.viewmodel
+
+import android.widget.Toast
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.lifecycle.ViewModel
+import org.sopt.at.screen.SignUpScreen
+
+class AuthViewModel : ViewModel() {
+
+    var registeredId = mutableStateOf("")
```

Comment:
뷰모델의 변수는 var로 두는 것 보다 아래 처럼 만들어봐도 좋을 것 같아요.
```suggestion
    private val _registeredId: MutableStateFlow<String> = MutableStateFlow("")
    val registeredId: StateFlow<String> = _registeredId.asStateFlow()
```
_registeredId 변수는 private로 설정되어 뷰모델 내에서만 값의 변경을 허용합니다.
registeredId는 screen에서 읽기 전용으로 사용될 변수입니다. 
이렇게 쓰기, 읽기를 구분해서 뷰모델 변수의 변경도 뷰모델 내에서 처리해봐도 좋을 것 같습니다.